### PR TITLE
lib: rstrcmp: null terminator check

### PR DESCRIPTION
**Pull request by mwierzbix**
Fix for issue mentioned @ https://github.com/thesofproject/sof/issues/198

### CODE REVIEW

File: src/lib/lib.c [hunk 1]
```diff
@@ -114,10 +114,15 @@ int rstrcmp(const char *s1, const char *s2)
 		if (*s1 < *s2)
 			return -1;
 		if (*s1 > *s2)
-			return 1;
+			return  1;
```
Comment (xiulipan) [inline src/lib/lib.c]: why add a space here.

Comment (mwierzbix) [reply src/lib/lib.c]: For human readability purposes. Is this against style guidelines of this project?

**Comment by mwierzbix** [comment]
Yes, haven't been sure what is the policy regarding this. Will squash (reasonably) from now on.

File: src/lib/lib.c (hunk 1 above)
Comment (mwierzbix) [reply src/lib/lib.c]: @lgirdwood @plbossart Is this an issue?

Comment (plbossart) [reply src/lib/lib.c]: This creates differences for no good reason. We typically try to stick to existing style/alignment unless there is a visible improvement, which isn't the case there.

Comment (mwierzbix) [reply src/lib/lib.c]: @plbossart Acknowledged. Will change.

**Comment by mwierzbix** [comment]
@jajanusz Squashed commits. @xiulipan @plbossart Removed unnecessary whitespace.

**Pull request merged**